README.md

```diff
@@ -1,2 +1,39 @@
-# scrapy-dsl
-DSL to simplify write spiders
+# scrapy-dsl (aka sdsl)
+
+Reading the book [Writing an interpreter in Go](https://interpreterbook.com/) i
+decided to build one very simple DSL to study and have fun :)
+
+## Proposal
+
+DSL to simplify write spiders. Propose this domain specific language is
+abstract python and scrapy knowledgement and provide one language more simple to write spiders.
+
+## DSL syntax proposal
+    
+1)     
+ 
+    get 'http://www.telelistas.net' extract phone on xpath('/div[@id=telInfo]');
+
+
+2) 
+    let firtBody = get('http://www.telelistas.net')
+    let results = parser(firstBody){
```

Comment:
Why would you need let here? :D

Reply:
we don't need :)
the first proposal is better.

Reply:
Actually if you have variables that can change value and closures in your language you will need something to disambiguate from altering an outer scope variable or creating a new variable. Example on a discussion about this can be found [here](https://github.com/NeowayLabs/nash/blob/master/proposal/1-scope-management.md).